### PR TITLE
Corrections to image metadata check CI

### PR DESCRIPTION
**Pull request by doofus-01**
Correcting some mistakes in [78a4c5c](https://github.com/wesnoth/wesnoth/commit/78a4c5c494ee2cd50f97347f52776d0a3c741c61)

### CODE REVIEW

File: .github/workflows/image_metadata.yml [hunk 1]
```diff
@@ -56,7 +56,7 @@ jobs:
             if [ -z "$copyright" ] ; then
               echo "no Copyright EXIF tag in $file"
               exit 1
-            elif [[ ( "$copyright" != ${accepted_cr[0]} ) && ( "$copyright" != ${accepted_cr[1]}) ]] ; then
+            elif [[ ! "|${accepted_cr[*]}|" =~ "|${copyright}|" ]]; then
```
Comment (soliton-) [inline .github/workflows/image_metadata.yml:59]: There is really no reason to write weird code like this.

File: .github/workflows/image_metadata.yml [hunk 2]
```diff
@@ -34,11 +34,11 @@ jobs:
 
       - name: read/export image metadata
         run: |
-          mapfile -t image_files < <(git diff --name-only ${BASE_SHA} ${SHA} | grep '\.webp$')
+          mapfile -t image_files < <(git diff --name-only ${BASE_SHA} ${SHA} | egrep '\.webp$|\.png$|\.jpg$|\.jpeg$')
```
Comment (soliton-) [inline .github/workflows/image_metadata.yml:37]:
```suggestion
          mapfile -t image_files < <(git diff --name-only "$BASE_SHA" "$SHA" | grep -E '\.(webp|png|je?pg)$'))
```

**Comment by doofus-01** [comment]
I shouldn't leave this lingering, even if it isn't perfect.

**Comment by soliton-** [comment]
You shouldn’t push broken code either. I even proposed working alternatives…

**Pull request merged**